examples/calling-apis/spa-and-hono-api/README.md

```diff
@@ -0,0 +1,228 @@
+# Chat SPA Application with Auth0 AI and AI SDK and Federated Connection Access Tokens
+
+The following app demonstrates using a SPA chatbot application, a backend API (and a linked API client), and a federated
+connection to a third party API.
+
+## Features
+
+This template leverages [bhvr](https://github.com/stevedylandev/bhvr) template for building a React SPA application with a Hono API.
+
+- **Full-Stack TypeScript**: End-to-end type safety between client and server
+- **Shared Types**: Common type definitions shared between client and server
+- **Monorepo Structure**: Organized as a workspaces-based monorepo with Turbo for build orchestration
+- **Modern Stack**:
+  - [Bun](https://bun.sh) as the JavaScript runtime and package manager
```

Comment:
^^ opting to remove `bun` in favor of `npm` for better repo consistency. Will add this in a future a commit (and personal reminder here 😄 )

Reply:
this was done in 52df8b229f0707e2902c81191c239f79e21ec28e, and should work 🥳 

Also took some time to update the logos and branding more towards Hono + React + Auth0.
<img width="1759" height="1191" alt="Screenshot 2025-08-07 at 5 39 05 PM" src="https://github.com/user-attachments/assets/be242ac4-a963-40a0-9bfc-5c2be9ac3b50" />